.circleci/config.yml

```diff
@@ -47,14 +47,14 @@ jobs:
     executor: docker-executor
     steps:
       - push:
-          comma_separated_tags: "1.10.5-alpine3.10,1.10.5-alpine"
+          comma_separated_tags: "1.10.5-alpine3.10,1.10.5-alpine,1.10.5,latest"
```

Comment:
@schnie are you going manually change this file every airflow release?

Reply:
For now, yea we'll have to update this when we add new versions.